Safety/out-of-hours-support.rst

```diff
@@ -39,17 +39,19 @@ Once enrolled, you can `download`_ a desktop application or browser extension to
     Your number will also be accessible to everyone with permissions to the vault.
     If you volunteer, please provide your availability to the best of your knowledge, including holdiays and weekends.
 
-.. _out-of-hours-support-procedure:
+.. _out-of-hours-support-resources-for-help:
 
-Procedure
-^^^^^^^^^
+Resources for help
+^^^^^^^^^^^^^^^^^^
 
 - If a fault happens during the night, or some new behavior appears in the system, observers and summit staff should attempt to troubleshoot to the extent of their knowledge.
 
 - Ask for help on Slack:
-    - `#auxtel-operations`_
-    - `#auxtel-virtual-control-room`_
-    - `#rubin-observing`_
+    - `#summit-auxtel`_
+    - `#summit-maintel`_
+    - `#summit-control-room`_
+
+Find :ref:`more information <Daytime-Nighttime-Slack-Channel-Communication-with-Summit-Personnel>` about Slack communications and the different channels being monitored.
 
 - If no response is received from Slack, and summit staff have run out of ideas or need assistance, consult the Emergency Call List.
 
```

Comment:
Can we add the link to the Confluence page where the list of volunteers is? 
There's also a typo on the note: "exhAUst the call list"

Reply:
We can't link to Confluence pages, but we can update when this is more official. 👍